### ISSUE TITLE
System.Net.NetworkInformation.NetworkInformationException in ServiceBrowser.StartBrowse() caused by non-IPv4 NICs

### ISSUE THREAD

**Issue by Oskar-Armbruster** [tmds/Tmds.MDns#14]
If one or more NICs have IPv4 disabled, an exception is thrown on ServiceBrowser.StartBrowse():
```
System.Net.NetworkInformation.NetworkInformationException (0x80004005)
   at System.Net.NetworkInformation.SystemIPInterfaceProperties.GetIPv4Properties()
   at Tmds.MDns.ServiceBrowser.CheckNetworkInterfaceStatuses(Dictionary`2 interfaceHandlers)
   at Tmds.MDns.ServiceBrowser.StartBrowsing(SynchronizationContext synchronizationContext)
   at Tmds.MDns.ServiceBrowser.StartBrowse(IEnumerable`1 serviceTypes, SynchronizationContext synchronizationContext)
   at Tmds.MDns.ServiceBrowser.StartBrowse(String serviceType, Boolean useSynchronizationContext)
```

System:
Windows 7 Professional Service Pack 1, 64bit
Visual Studio Professional 2015 Update 3
.NET Framework 4.7.1
Tmds.MDns 0.6.0 via NuGet

**Comment by tmds** [comment]
Probably we need a check like
```
networkInterface.Supports(NetworkInterfaceComponent.IPv4)
```

with these checks:

https://github.com/tmds/Tmds.MDns/blob/21007010f355fe54f1e8f4785dd5c8cf49f6bbcc/src/Tmds.MDns/ServiceBrowser.cs#L240-L251


@Oskar-Armbruster can you try and implement this and send a PR? If not, I can look at this, probably next week.

**Comment by Oskar-Armbruster** [comment]
Thanks for the quick reply! Your suggestion works perfectly. I added `if (!networkInterface.Supports(NetworkInterfaceComponent.IPv4)) continue;` on line 253 of ServiceBrowser.cs and the exception is not thrown anymore. I tested it on a machine with 1 NIC and IPv4 disabled. Some time next week I will be able to try it on a multi NIC machine. I will let you know what came out as soon as possible. Thanks a lot, Oskar
